org.eclipse.lsp4j.jsonrpc.debug/src/main/java/org/eclipse/lsp4j/jsonrpc/debug/DebugRemoteEndpoint.java

```diff
@@ -34,7 +34,7 @@ public DebugRemoteEndpoint(MessageConsumer out, Endpoint localEndpoint,
 	@Override
 	protected DebugRequestMessage createRequestMessage(String method, Object parameter) {
 		DebugRequestMessage requestMessage = new DebugRequestMessage();
-		requestMessage.setId(String.valueOf(nextSeqId.incrementAndGet()));
+		requestMessage.setId(nextSeqId.incrementAndGet());
```

Comment:
This resolves one of the annoying issues I had, I really didn't like the converting back and forth, so thanks for this.